Content.Server/Lightning/LightningSystem.cs

```diff
@@ -37,28 +37,28 @@ private void OnRemove(EntityUid uid, LightningComponent component, ComponentRemo
 
     private void OnCollide(EntityUid uid, LightningComponent component, ref StartCollideEvent args)
     {
-        if (!TryComp<BeamComponent>(uid, out var lightningBeam) || !TryComp<BeamComponent>(lightningBeam.VirtualBeamController, out var beamController))
```

Comment:
I'm not entirely sure what the function was meant to be doing, but AFAICT there was a chance it would enter an infinite loop if `Arc` failed to find a target? So now it no longer does a while loop, which might change some behaviour if it fails to find a target during the first attempt.